Erupe/server/signserver/dsgn_resp.go

```diff
@@ -77,13 +79,19 @@ func (s *Session) makeSignInResp(uid int) []byte {
 		} else {
 			bf.WriteUint16(char.HRP)
 		}
+
+		t := japanese.ShiftJIS.NewEncoder()
+		str_name, _, err := transform.String(t, char.Name)
+		if err != nil {
+		}
```

Comment:
```suggestion
		if err != nil {
		  str_name = char.Name
		}
```

handle the error here, otherwise, it will return an empty name